terraform/s3.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_s3_bucket" "c12-court-s3" {
+  bucket = "c12-court-bucket"
+
+  force_destroy = true
+
+  tags = {
```

Comment:
Are the tags necessary here? What do they do?

Reply:
Not necessary but, I thought it was a good practice given identifying the s3 was crucial.